### PR TITLE
[MI14]:Removed useless and unwanted curly braces around statement.

### PR DESCRIPTION
**Pull request by AbhinavRamKola**
**What's in the PR**
Unwanted curly braces are removed  because of unnecessary use of them in lambda
functions. When there is only one statement inside the lambda function
then there is no need for the curly braces.

### CODE REVIEW

File: inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java [hunk 1]
```diff
@@ -155,9 +155,9 @@ public String renderHtml()
         }
 
         // Sort backwards
-        nodes.sort((a, b) -> {
-            return b.position - a.position;
-        });
+        nodes.sort((a, b) -> 
+             b.position - a.position
+        );  //30
```
Comment (munterkalmsteiner) [inline inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java]: What does the //30 mean here?

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
Looks good, except the additional comments (numbers) that were introduced.

File: inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java (hunk 1 above)
Comment (AbhinavRamKola) [reply inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java]: I added these comments to track my tasks. It does not say anything about the changes i made. I will remove them.

Comment (AbhinavRamKola) [reply inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java]: I added these comments while tracking my tasks. They are not relevant to the changes that I made. I shall remove them.

**Review by AbhinavRamKola — COMMENTED** [review]
I have removed all the unwanted comments.

**Review by munterkalmsteiner — APPROVED** [review]
Looks good!

**Pull request closed without merging**